examples/guestbook/README.md

```diff
@@ -67,7 +67,82 @@ The web front end interacts with the redis master via javascript redis API calls
 
 This example requires a running Kubernetes cluster.  See the [Getting Started guides](../../docs/getting-started-guides/) for how to get started. As noted above, if you have a Google Container Engine cluster set up, go [here](https://cloud.google.com/container-engine/docs/tutorials/guestbook) instead.
```

Comment:
Also link to https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/prereqs.md